internal/ini/visitor.go

```diff
@@ -125,8 +164,27 @@ func (t Sections) List() []string {
 // Section contains a name and values. This represent
 // a sectioned entry in a configuration file.
 type Section struct {
-	Name   string
+	// Name is the Section profile name
+	Name string
+
+	// values are the values within parsed profile
 	values values
+
+	// Errors is the list of errors
+	Errors []error
+
+	// Logs is the list of logs
+	Logs []string
+
+	// SourceFile is the INI Source file from where this section
+	// was retrieved.
+	SourceFile string
```

Comment:
This probably should be a list of strings. But not clear how that will work. Coz if a reference is no longer relevant after merging multiple files, we will have to remove it?